src/_P016_IR.ino

```diff
@@ -235,7 +242,7 @@ boolean Plugin_016(uint8_t function, struct EventStruct *event, String& string)
 
         uint16_t bufsize = P016_BUFFERSIZE;
 
-        if ((bufsize < 100) || (bufsize > 1024)) { bufsize = P016_DEFAULT_BUFFERSIZE; } // safety check
+        if ((bufsize < P016_MIN_BUFFERSIZE) || (bufsize > P016_MAX_BUFFERSIZE)) { bufsize = P016_DEFAULT_BUFFERSIZE; } // safety check
```

Comment:
Since you did all this effort, perhaps you should bind the minimum buffer size to if AC detailed support is used or not.

Reply:
What typical size is needed for AC remotes?

Reply:
The buffer size of 100 (uint16_t's) already can be 'problematic' when using a simple tv-remote, because of the multiple repetitions of that remote (it seems to send 2 to 5 times the same signal, even on a short press). That's why I've set the default to 150 (300 bytes), as that was enough for my remote. The library suggested default was 100, so I took that as the minimum value. Don't think the default needs to be increased just because the AC feature is included, as that is always included in the regular IR builds, so everyone would be using the larger minimum default, even when not needed. I've tried to make the documentation clear enough for users to understand when/why to increase this buffer size, so they have the option to set it to the max. defined, it will just cause rather high memory usage of the plugin.
If you think a more elaborate explanation is needed, please suggest what to add (in the documentation) 👍